taskcluster/ci/browsertime/kind.yml

```diff
@@ -232,3 +232,38 @@ jobs:
         test-name: google-search-restaurants
         treeherder:
             symbol: 'Btime(tp6m-28-c)'
+
+    youtube-playback:
+        test-name: browsertime-youtube-playback
+        treeherder:
+            symbol: 'Btime(ytp)'
```

Comment:
You need to add the args from here to this task: https://github.com/mozilla-mobile/fenix/blob/master/taskcluster/ci/raptor/kind.yml#L93-L98